pkg/network/protocols/http/testutil/testutil.go

```diff
@@ -91,11 +127,16 @@ func HTTPServer(t *testing.T, addr string, options Options) func() {
 			return err
 		}
 	}
-	err := listenFn()
-	if err != nil {
+
+	if err := listenFn(); err != nil {
 		t.Fatalf("server listen: %s", err)
 	}
-	return func() { srv.Shutdown(context.Background()) }
+	return func() {
+		srv.Shutdown(context.Background())
+		if options.EnableTCPTimestamp != nil {
+			setNetIPV4TCPTimestamp(t, oldTCPTS)
+		}
```

Comment:
in favor of https://github.com/DataDog/datadog-agent/pull/13311#discussion_r1190389529
```suggestion

```

Reply:
done